examples/00-EDB/15_ac_analysis.py

```diff
@@ -1,50 +1,53 @@
 """
-EDB: SYZ analysis
--------------------
-This example shows how you can use PyAEDT to set up SYZ analysis on Serdes channel.
-The input is the name of the differential nets. The positive net is PCIe_Gen4_TX3_CAP_P.
-The negative net is PCIe_Gen4_TX3_CAP_N. The code will place ports on driver and
+# EDB: Network Analysis in SIwave
+
+This example shows how to use PyAEDT to set up SYZ analysis on a 
+[serdes](https://en.wikipedia.org/wiki/SerDes) channel.
+The signal input is applied differetially. The positive net is _"PCIe_Gen4_TX3_CAP_P"_.
+The negative net is _"PCIe_Gen4_TX3_CAP_N"_. In this example, ports are placed on the
```

Comment:
```suggestion
# EDB: Network Analysis in SIwave

This example shows how to use PyAEDT to set up SYZ analysis on a 
[serdes](https://en.wikipedia.org/wiki/SerDes) channel.
The signal input is applied differetially. The positive net is _"PCIe_Gen4_TX3_CAP_P"_.
The negative net is _"PCIe_Gen4_TX3_CAP_N"_. In this example, ports are placed on the
```
```suggestion
# EDB: Network analysis in SIwave

This example shows how to use PyAEDT to set up an SYZ analysis on a 
[SerDes](https://en.wikipedia.org/wiki/SerDes) channel.
The signal input is applied differentially. The positive net is _``PCIe_Gen4_TX3_CAP_P``_.
The negative net is _``PCIe_Gen4_TX3_CAP_N``_. In this example, ports are placed on the
```
I'm not sure what there are underscores before and after the PCle code entities. While you used quotation marks, I THINK these should be code entities (with no underscores before and after the names).